create/approve.py

```diff
@@ -21,10 +22,10 @@
 
 TEMPLATE = dedent(
     """\
-    user_name: {empty}
+    user_name:
     group_name: {group_name}
     callink_oid: {callink_oid}
-    signatory: {empty}
+    signatory:
```

Comment:
If you do this, `group_name`, `callink_oid`, and `email` will have trailing spaces when the editor opens up even if no OID is passed in, but `signatory` and `user_name` will not. I know there have been complaints that the YAML parser will fail if you accidentally leave out the space after the colon, so it seemed like a general improvement in addition to simple consistency.